### ISSUE TITLE
Hey Jadu Fellow - Get in touch on discord

### ISSUE THREAD

**Issue by arslan-a1i** [RadikaPremChand/bootcamp2020#1]
### Hi @RadikaPremChand 

I'm Arslan Ali, your fellow tribe-09 member for **Jadu Fellowship Program**.  I am writing this here because I couldn't find any other way to get in touch. We are having trouble with connecting a few fellow-members, so thought maybe I should ask if you are having some issues with the discord server. 

**Please let me know if there are any issues with that and we will try to resolve them to stay connected with the other fellows and the Jadu Team.**

**Comment by RadikaPremChand** [comment]
Hi Arslan,

Are you using discord through web or app?

Best,

Radika

**Comment by arslan-a1i** [comment]
I am using the desktop app. The web app is also active in my browser.

**Comment by RadikaPremChand** [comment]
Okay, I check discord randomly. Do you know the other tribe members or we
can have a WhatsApp group to know each other?

**Comment by arslan-a1i** [comment]
Yep, two other members are in touch with me on Discord. Maybe you should give it try as most of our work activities are shared on that discord server. I am not a Whatsapp user myself. But can't say about the others...

**Comment by arslan-a1i** [comment]
### **Hi @RadikaPremChand. Hope you are doing well.**

Well, you might have noticed that **we have a group project which is due on the 31st of March**. As it will be a collective effort of the tribe, so we were having a discussion on how to manage all this stuff.

We will be having a detailed discussion tomorrow and then after that, we will start working on it.

Hope you would also like to join, cos this is a group project, not the one that we can do on an individual basis.
So, here's the link to our Whatsapp Group, if you would like to join. :)

[**Whatsapp Group: Tribe-9**](https://chat.whatsapp.com/BNIw2ZhjPriHRnbacoicQN?fbclid=IwAR1oK2HIzeCUKr76LyW-HrySlKzW31os_dopiN8XnJ5ESO2zXFe3zMF1vWM)
